### PR TITLE
Calc waveforms from kilosort output data

### PR DESCRIPTION
**Pull request by resatomo**
Get waveforms and unit positions from raw data in neuropixels, using spike indices of each unit obtained by kilosort

### CODE REVIEW

**Comment by rcanfield2** [comment]
Does this account for drift correction? It doesn't look like it does

**Comment by resatomo** [comment]
In my understanding, spike times and their clusters already result from drift correction. It computes drift map based on spike amplitudes, and correct drift at each batch. Then it computes spikes times again.

**Comment by resatomo** [comment]
I got what you meant now. Spike times are affected by drift correction. But channel mapping files are static over time. Do you think we need waveforms even after drift?

File: aopy/precondition/base.py [hunk 1]
```diff
@@ -598,6 +598,62 @@ def bin_spikes(data, fs, bin_width):
     binned_spikes = binned_spikes/bin_width # convert from [spikes/bin] to [spikes/s]    
     return binned_spikes
 
+def calc_ks_waveforms(raw_data, sample_rate, spike_times_unit, templates, channel_pos, waveforms_nch=10, time_before=1000., time_after=1000.):
+    '''
+    Calculate waveforms, waveform channels, and positions of units, using templates from kilosort
+
```
Comment (rcanfield2) [inline aopy/precondition/base.py:604]: Can you add a comment here that this function does not account for drift correction?

**Review by rcanfield2 — APPROVED** [review]

**Pull request merged**